swagger-doclet/src/main/java/com/tenxerconsulting/swagger/doclet/parser/ParserHelper.java

```diff
@@ -1105,10 +1176,21 @@ private static boolean isAssignableFrom(ClassDoc superClass, ClassDoc clazz) {
      */
     public static List<String> getConsumes(MethodDoc methodDoc, DocletOptions options) {
         List<String> methodLevel = listInheritableValues(methodDoc, JAX_RS_CONSUMES, "value", options);
+        
+        if (methodLevel == null) {
+            methodLevel = listInheritableValues(methodDoc, SPRING_MVC_REQUEST_MAPPING, "consumes", options);
+        }
+        
+        if (methodLevel == null) {
+            // look for class level
+            methodLevel = listValues(methodDoc.containingClass(), JAX_RS_CONSUMES, "value", options);
```

Comment:
Is this fixing a bug in the current JAXRS annotation processing?

Reply:
No, the JAXRS processing should remain the same. I have just added the Spring annotations, but plz correct me, if I'm wrong.